components/nav.php

```diff
@@ -10,15 +10,38 @@
 	}
 ?>
 
-<div class='section theme-dark nav'>
-	<div class='sub-section-full'>
-		<div class='wrap-left menu-icon-wrap'>
-			<i class='menu-icon material-icons'>menu</i>
-			<!-- <i class='menu-icon material-icons'>&#xE5D2;</i> -->
-		</div>
-		<div class='wrap nav-title-section'>
-			<div id='page-title'><?php echo $page_title ?></div>
-			<div id='page-subtitle'><?php echo $page_subtitle ?></div>
+<nav class="navbar navbar-expand-md navbar-dark bg-dark sticky-top">
+	<div class="container">
+		<a class="navbar-brand" href="/"><?php echo $page_title ?></a>
+
+		<button class="navbar-toggler border-0" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+
+		<div class="collapse navbar-collapse" id="navbarsExampleDefault">
+			<span class="navbar-text d-none d-md-inline mr-auto">
+				<?php echo $page_subtitle ?>
+			</span>
+
+			<ul class="navbar-nav">
+				<li class="nav-item">
+					<a class="nav-link" href="/queue">Queue</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="/login">Volunteer Login</a>
+				</li>
+			<?php if (isset($_REQUEST['admin'])): ?>
```

Comment:
First of all, this is temporary. Throw `?admin` onto the url to see the additional menus.